packages/contracts/src/manager/access/AccessControl.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import {IAccessControl} from "../interfaces/IAccessControl.sol";
+import {AccessControlInternal} from "./AccessControlInternal.sol";
+
+/**
+ * @title Role-based access control system
+ * @dev derived from https://github.com/OpenZeppelin/openzeppelin-contracts (MIT license)
+ * @dev https://github.com/solidstate-network/solidstate-solidity/blob/master/contracts/access/access_control/AccessControl.sol
+ */
+abstract contract AccessControl is IAccessControl, AccessControlInternal {
```

Comment:
Unless we forking and maintaining our own version of these contracts then they should just be imported from lib/open-zeppelin. There's no reason to copy them here.

Reply:
> Unless we forking and maintaining our own version of these contracts then they should just be imported from lib/open-zeppelin. There's no reason to copy them here.

Access control system on OZ can't be used for diamond implementation